components/form-builder/app/responses/DownloadTableDialog.tsx

```diff
@@ -50,6 +56,7 @@ export const DownloadTableDialog = ({
   unknownErrorTitle,
   unknownErrorDescription,
   unknownErrorDescriptionLink,
+  tempType,
```

Comment:
This will be refactored into two dialogs in the next PR along with the below code.